src/main/java/com/google/sps/data/DateTimeRange.java

```diff
@@ -61,4 +68,20 @@ public String getEndDate() {
   public String getTime() {
     return isTimeSet() ? startTime : null;
   }
+
+  public String getStartTime() {
+    return startTime;
+  }
+
+  public String getEndTime() {
+    return endTime;
+  }
+
+  public String convertStartTimeToString() {
```

Comment:
Optional: Maybe make it clear that it is in UTC, e.g. name it `convertStartTimeToUTCString`